pkg/plugins/policies/meshhealthcheck/plugin/v1alpha1/plugin.go

```diff
@@ -158,6 +166,39 @@ func configure(
 	return nil
 }
 
+func applyToEgressRealResources(rs *core_xds.ResourceSet, proxy *core_xds.Proxy) error {
+	for _, resource := range proxy.ZoneEgressProxy.MeshResourcesList {
```

Comment:
naming is a bit confusing in this function, I'd call `meshResources` instead of just `resource`